components/card/card.css

```diff
@@ -1,70 +1,150 @@
 .card {
-  height: 350px;
-  width: 300px;
-  border: 1px solid gray;
+  background-color: #fff;
   display: flex;
-  align-items: center;
-  justify-content: center;
-  margin: 0 20px;
-  border-radius: 5px;
+  flex-direction: column;
+  width: 350px;
+  min-height: max-content;
+  margin: 10px auto;
+  border: 1px solid gray;
+  border-radius: 15px;
 }
 
-.card-badge,
-.card-dismiss {
+.card__top {
+  display: flex;
```

Comment:
You are using flex in quite a lot of places so why not make a utility class for flex itself, it will make the code cleaner.